### PR TITLE
String input

### PR DESCRIPTION
**Pull request by TDW89**
This is the on screen keyboard I was working on a while back with all the bugs fixed (i think).
there is also a small dependency checker function used by the keyboard to allow it to run with or without `lib_spec_char.ksm` and by the example script (a login interface) to determine if you have previously created an account and deny you access to the terminal if you get the user name or password wrong (when used as a boot script).

### CODE REVIEW

File: examples/example_lib_input_string.ks [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+run lib_input_string.
+run lib_file_exists.
+run spec_char.
+
+if not file_exists("acount.ksm") {
+ set newuser to true.
+} else {
+ set newuser to false.
+ run acount.ksm.
```
Comment (space-is-hard) [inline examples/example_lib_input_string.ks]: *account

Not sure if this relies on a specific script; if so, this is a minor spelling error that can be ignored

Comment (TDW89) [inline examples/example_lib_input_string.ks]: nah its a quick fix, its a file generated by the script so all references to it are contained to that script. will fix it now.

**Comment by TDW89** [comment]
Ok fixed and retested (since i was modifying the script).

**Comment by space-is-hard** [comment]
Looks good from here; I'll let you merge this one if you feel it's ready

**Pull request merged**